box-office.rb

```diff
@@ -1,5 +1,12 @@
 require 'sinatra'
+require 'httparty'
 
 get '/' do
   'Hello world!'
 end
+
+get '/movies/:imdbid' do |id|
+  response = HTTParty.get("http://www.omdbapi.com/?apikey=#{ENV['OMDB_API_KEY']}&i=#{id}&plot=full&r=json")
```

Comment:
I'm not sure if we need to add another route just yet. The Hello world route OK. But if we do TDD, we need to add a test framework (1 PR), add the test for the new route and add the route (could be 2 or 1 PRs? I don't know.). If we're not going to do TDD, it could be chaotic but also fun.